keanu-python/tests/test_base.py

```diff
@@ -0,0 +1,80 @@
+import keanu as kn
+import numpy as np
+import pytest
+from py4j.java_gateway import java_import
+
+jvm_view = kn.KeanuContext().jvm_view()
+java_import(jvm_view, "io.improbable.keanu.vertices.dbl.probabilistic.GaussianVertex")
+
+
+def test_can_pass_scalar_to_vertex():
+    gaussian = kn.Vertex(jvm_view.GaussianVertex, (0., 1.))
```

Comment:
I think you should implement `kn.Gaussian`. The user is never going to create a vertex explicitly with the `jvm_view` like this, are they?

Reply:
No, they won't. I'm just testing the vertex class to make sure it works